docs/modules/ROOT/pages/installation/knative.adoc

```diff
@@ -0,0 +1,35 @@
+= Knative installation
+
+"https://knative.dev[Knative] is an Open-Source Enterprise-level solution to build Serverless and Event Driven Applications. The effort done in this project is a great complement to Camel K, which can leverage natively some feature offered by Knative. In particular, Camel K will be able to leverage "scale to 0" (hence, serverless) feature offered by Knative.
+
+NOTE: Knative is an optional configuration. It is not required to run Camel K.
+
+[[install-knative]]
+== Knative roles configuration
+
+Camel K needs to have certain privileges to use the resources used by Knative. We have identified them, so, it follows a few configuration that you will need to apply in order to make Camel K work with Knative.
+
+NOTE: We assume you have already both the Knative operator and the Camel K operator up and running. If not, the first step is to install both operator. See the guide on xref:installation/installation.adoc[how to install Camel K] and the guide on https://knative.dev/docs/install/[how to install Knative] respectively.
+
+Once all the resources are installed in the cluster, you'll need to perform the following action:
+
+```bash
+kubectl apply -k https://github.com/apache/camel-k/tree/main/config/rbac/knative?ref=v2.1.0
+```
+
+The configuration above is using the released version `2.1.0`. You may want to use instead another version or the latest development available on `main` branch. Before continuing with the configuration, you need to make sure to know which are the namespaces on which the Operator is running and on which your Integration is going to run (they could be the same). In order to change it, you must perform some customization locally in a similar fashion:
+
+```bash
+git clone https://github.com/apache/camel-k.git
+cd camel-k/config/rbac/knative
+sed -i "s/namespace: placeholder/namespace: $OPERATOR_NAMESPACE/" operator-cluster-role-binding-custom-resource-definitions.yaml
```

Comment:
From what I understand of kustomize, replacing "placeholder" by "default" in the rbac yaml files. This should also works on [subjects](https://github.com/kubernetes-sigs/kustomize/blob/e7c8ed18998745680cf45efd12e5990cedcb3d39/api/filters/namespace/namespace.go#L108-L125).

Reply:
That would be great as it simplify the procedure a lot. However I wonder how it works when the operator is installed in a global namespace. I'll make some test to see if it is the proper solution, thanks for the hints.

Reply:
The only issue I had was when one of the resource need to be on another namespace that the one declared with `kustomize edit set namespace $IT_NAMESPACE` (in my case I want to keep the `camel-k-operator-local-registry` RoleBinding on namespace `kube-public`).
What is the specifics of a "global" namespace ?

Reply:
Basically the operator could be installed in any namespace, ie `global-ns`. The service account `camel-k-operator` is already existing on that namespace and we must instruct the kustomize procedure that this is the namespaced resource we want to provide as a subject for the RoleBinding. I've tried to do some test, but I could not make it work without explicitly setting the namespace replacement :/